README.md

```diff
@@ -1,51 +1,21 @@
 # Enumerable methods rebuild
 
-> One paragraph statement about the project.
-
-![screenshot](./app_screenshot.png)
-
-Additional description about the project and its features.
+> This project goal was to rebuild some Ruby's built-in Enumerables methods, each, each_with_index, select, any?, all?, none? count, map and inject. 
 
 ## Built With
 
-- Major languages,
-- frameworks,
-- technologies used
-
-## Live Demo
-
-[Live Demo Link](https://livedemo.com)
-
+- Ruby 2.6
 
 ## Getting Started
 
```

Comment:
You should include in your ReadMe the instructions to use your code in the `irb`, and then, when you include the tests, you can add that, too.